src/android/index.jsx

```diff
@@ -261,24 +262,43 @@ class Android extends Component {
                           eq: {
                             framework: 10,
                             repo: 'mozilla-central',
-                            suite: 'raptor-speedometer-geckoview',
+                            suite: 'raptor-speedometer-geckoview-power',
                           },
                         },
                       ],
                     },
                   },
                   {
-                    label: 'Pixel 2 (ARM64)',
+                    label: 'Firefox Preview',
                     filter: {
                       and: [
                         { missing: 'test' },
+                        {
+                          prefix: { platform: 'android-hw-g5-7-0-arm7-api-16' },
+                        },
+                        {
+                          eq: {
+                            framework: 10,
+                            repo: 'mozilla-central',
+                            suite: 'raptor-speedometer-fenix-power',
+                          },
+                        },
+                      ],
+                    },
+                  },
+                  {
+                    label: 'Firefox ESR',
+                    filter: {
+                      and: [
+                        { missing: 'test' },
+                        {
+                          prefix: { platform: 'android-hw-g5-7-0-arm7-api-16' },
+                        },
                         {
                           eq: {
                             framework: 10,
-                            options: 'opt',
-                            platform: 'android-hw-p2-8-0-android-aarch64',
                             repo: 'mozilla-central',
-                            suite: 'raptor-speedometer-geckoview',
+                            suite: 'raptor-speedometer-fennec68-power',
```

Comment:
The `-power` is the same test while measuring power usage.  Do these three browsers have no-power versions?

Reply:
Yes, they have no-power versions.

Reply:
I have replaced it with no-power version.

Reply:
I couldn't find `BENCHMARK_SUITES` config variable. Generating `series ` with this seems more elegant than what I did.